products/vmware-horizon.md

```diff
@@ -0,0 +1,57 @@
+---
+title: VMware Horizon
+permalink: /vmware-horizon
+# Since we don't have any other VMware products as of now
```

Comment:
now we have :) 
so lets change /vmware to /vmwarehorizon

Reply:
I prefer the `-` in the URL makes it more readable.